spec/routing/work_packages_spec.rb

```diff
@@ -66,10 +64,10 @@
     end
   end
 
-  it 'should connect GET /work_packages/:id to work_packages#show' do
+  it 'should connect GET /work_packages/:id to work_packages#index' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.